src/main/java/Duke.java

```diff
@@ -1,10 +1,276 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+class Task {
```

Comment:
Would it make the whole project more well-organised if these extra classes are written in separated files?